### PR TITLE
Prepare for 3.5.0 release

### PR DESCRIPTION
**Pull request by annakhm**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,88 @@
...
+Fabric support and user management support are offered with this release. For detailed list of supported objects, please see below.
+
+* `data/nsxt_compute_collection`
+* `data/nsxt_compute_manager`
+* `data/nsxt_failure_domain`
+* `data/nsxt_policy_uplink_host_switch_profile`
+* `data/nsxt_failure_domain`
+* `data/nsxt_manager_cluster_node`
+* `data/nsxt_policy_host_transport_node_profile`
+* `data/nsxt_transport_node`
```
Comment (ksamoray) [inline CHANGELOG.md:13]: Replaced by data/nsxt_edge_transport_node

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,88 @@
...
+* `resource/nsxt_edge_cluster`
+* `resource/nsxt_compute_manager`
+* `resource/nsxt_manager_cluster`
+* `resource/nsxt_cluster_virtual_ip`
+* `resource/nsxt_edge_cluster`
+* `resource/nsxt_policy_uplink_host_switch_profile`
+* `resource/nsxt_failure_domain`
+* `resource/nsxt_policy_host_transport_node_profile`
+* `resource/nsxt_policy_transport_zone`
+* `resource/nsxt_transport_node`
```
Comment (ksamoray) [inline CHANGELOG.md:28]: Replaced by resource/nsxt_edge_transport_node

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,88 @@
...
+* `resource/nsxt_policy_tier1_gateway`: Respect ha_mode value when NSX auto-assigns it, to avoid non-empty diffs in the provider ([#1042](https://github.com/vmware/terraform-provider-nsxt/pull/1042))
+* `resource/nsxt_policy_tier0_gateway`, `resource/nsxt_policy_tier1_gateway`: Allow locking gateway locale_service by specifying nsx_id. This fixes the issue of dependant interfacing being deleted after applying a change in locale service ([#1040](https://github.com/vmware/terraform-provider-nsxt/pull/1040))
+* `resource/nsxt_policy_lb_pool`: Allow to specify multiple active monitor paths by adding new attribute and deprecating the old one ([#1039](https://github.com/vmware/terraform-provider-nsxt/pull/1039))
+* `data/nsxt_policy_tier0_gateway`, `data/nsxt_policy_edge_cluster`: Fix issue that prevented tenant user to accessing these data sources ([#1015](https://github.com/vmware/terraform-provider-nsxt/pull/1015))
+
+EXPERIMENTAL FEATURES:
+Upgrade support is offered as Beta with this release. Please note that upgrade feature requires a separate terraform config and state.
+Additional Beta features include separate security policy rule resource, LB profiles and GRE tunnel.
```
Comment (2ez4szliu) [inline CHANGELOG.md:58]: maybe we want to clarify that it's the policy version of LB profile resource

**Review by GraysonWu — COMMENTED** [review]
LGTM. I will defer to others to approve it.

File: website/docs/r/policy_security_policy_rule.html.markdown [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-subcategory: "Firewall"
+subcategory: "Beta"
```
[the remark this answers is not in the document]
Comment (annakhm) [reply website/docs/r/policy_security_policy_rule.html.markdown:2]: Yes

**Review by salv-orlando — APPROVED** [review]
LGTM

**Review by 2ez4szliu — APPROVED** [review]

**Review by GraysonWu — APPROVED** [review]

**Review by ksamoray — APPROVED** [review]

**Pull request merged**